.eslintrc

```diff
@@ -0,0 +1,77 @@
+{
+    "root": true,
+    "parser": "babel-eslint",
+    "extends": "airbnb",
+    "plugins": ["json", "prettier", "import", "jsx-a11y", "react"],
```

Comment:
Everything in this file exactly matches what was in `package.json`, with exception of adding the eslint prettier plugin

Reply:
Using a `.eslintrc` is advantageous over `package.json` configuration because it's more discoverable to other utilities that want to leverage (like prettier)